### PR TITLE
[HELM] add serviceAccount helm chart

### PR DESCRIPTION
**Pull request by zhaohehuhu**
### What changes were proposed in this pull request?

as title

### Why are the changes needed?

Support custom serviceAccount to control what permissions and resources a pod has access to.

### Does this PR introduce _any_ user-facing change?

No

### How was this patch tested?

 test the template rendering by helm template command line

### CODE REVIEW

File: charts/celeborn/templates/serviceaccount.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+#
+
+{{- if .Values.serviceAccount.create }}
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: {{ include "celeborn.serviceAccountName" . }}
+  labels:
+    {{- include "celeborn.labels" . | nindent 4 }}
+{{- end }}
```
[the remark this answers is not in the document]
Comment (zhaohehuhu) [reply charts/celeborn/templates/serviceaccount.yaml]: Done. Thanks.

File: charts/celeborn/values.yaml [hunk 2]
```diff
@@ -247,3 +247,9 @@ podMonitor:
     interval: 5s
     # Specifies port name
     portName: metrics
+
+serviceAccount:
+  # -- Specifies whether to create a service account for Celeborn
+  create: false
+  # -- Specifies the name of the service account
+  name: default
```
[the remark this answers is not in the document]
Comment (zhaohehuhu) [reply charts/celeborn/values.yaml]: Done. Thanks.

**Review by FMX — APPROVED** [review]
LGTM.

**Comment by FMX** [comment]
Merged into main(v0.6.0).

**Pull request closed without merging**